packages/polymath-issuer/src/utils/parsers/index.js

```diff
@@ -83,68 +83,89 @@ const checkColumnCount = data => {
   return errorMsg;
 };
 
+const checkCSVHeaders = data => {
+  // Split the input into lines
+  let rows = data.split('\n');
+  const header_row = rows[0].split(',');
+  const default_headers =
+    'Address,Sale Lockup,Purchase Lockup,KYC/AML Expiry,Can Buy From STO,Exempt From % Ownership';
```

Comment:
Is Exempt from % ownership a default header? The user may not have the max % ownership enabled.

Reply:
Yes it's default. It'll be empty if not enabled.